src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -133,6 +132,10 @@
             "com.puppycrawl.tools.checkstyle.checks.coding.VariableDeclarationUsageDistanceCheck",
             "com.puppycrawl.tools.checkstyle.checks.imports.ImportControlCheck"));
 
+    /** Inlined config is not supported for non java files. */
+    private static final Set<String> PERMANENT_SUPPRESSED_CHECKS = new HashSet<>(Arrays.asList(
+            "com.puppycrawl.tools.checkstyle.checks.OrderedPropertiesCheck"));
```

Comment:
As this collection is general, let move comment that we have now over field to be over Check name, and class field should have general comment that some Checks not about java so we can not use inlined configs. Inlined config is java style multiline comment.

In such permanent suppression, will be more Checks eventually